lte/gateway/c/oai/tasks/amf/amf_as.cpp

```diff
@@ -1164,11 +1164,10 @@ static int amf_as_security_req(
         if (ue_context) {
           amf_security_context_t* amf_security_context =
               &ue_context->amf_context._security;
-          amf_security_context->selected_algorithms.integrity = 0;
-          //   M5G_NAS_SECURITY_ALGORITHMS_128_5G_IA2;  // TODO get this
-          //   computed
-          amf_security_context->selected_algorithms.encryption = 0;
-          //  M5G_NAS_SECURITY_ALGORITHMS_5G_EA0;  // TODO get this computed
```

Comment:
Many places security header type hard coded with "SECURITY_HEADER_TYPE_INTEGRITY_PROTECTED_CYPHERED". can we please take of this as well